README.md

```diff
@@ -3,29 +3,34 @@
-| `futureProvider`               | Creates a FutureProvider                                                    |
-| `futureProviderFamily`         | Creates a FutureProvider with the family modifier                           |
-| `streamProvider`               | Creates a StreamProvider                                                    |
-| `streamProviderFamily`         | Creates a StreamProvider with the family modifier                           |
-| `changeNotifierProvider`       | Creates a ChangeNotifierProvider                                           |
-| `changeNotifierProviderFamily` | Creates a ChangeNotifierProvider with the family modifier                  |
-| `stateProvider`                | Creates a StateProvider                                                     |
-| `stateProviderFamily`          | Creates a StateProvider with the family modifier                            |
-| `stateProviderFamily`          | Creates a StateProvider with the family modifier                            |
-| `stateNotifierProvider`        | Creates a StateNotifier provider                                            |
-| `stateNotifierProviderFamily`  | Creates a StateNotifierProvider with the family modifier                   |
-| `scopedProvider`               | Creates a ScopedProvider                                                    |
-| `stateNotifier`                | Creates a class that extends StateNotifier and allows you to edit the types |
-
+| `futureProvider`               | Creates a Future provider                                                    |
+| `futureProviderFamily`         | Creates a Future Provider with the family modifier                           |
+| `streamProvider`               | Creates a Stream provider                                                    |
+| `streamProviderFamily`         | Creates a Stream Provider with the family modifier                           |
+| `changeNotifierProvider`       | Creates a Change Notifier provider                                           |
+| `changeNotifierProviderFamily` | Creates a Change Notifier Provider with the family modifier                  |
+| `stateProvider`                | Creates a State provider                                                     |
+| `stateProviderFamily`          | Creates a State Provider with the family modifier                            |
+| `stateProviderFamily`          | Creates a State Provider with the family modifier                            |
+| `stateNotifierProvider`        | Creates a State Notifier provider                                            |
+| `stateNotifierProviderFamily`  | Creates a State Notifier Provider with the family modifier                   |
+| `scopedProvider`               | Creates a Scoped provider                                                    |
+| `stateNotifier`                | Creates a class that extends State Notifier and allows you to edit the types |
```

Comment:
These seems to have changed as well, keep them as they were before as it makes more sense with the class names 😊

Reply:
I have not changed anything I think this is because of the merge conflict that occurred before.

Reply:
Could you change it back?